client/src/components/RatingsReviews/ReviewComponent.jsx

```diff
@@ -1,20 +1,52 @@
 /* eslint-disable indent */
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import moment from 'moment';
+import actions from '../../state/actions/index.js';
+import axios from 'axios';
+import AtelierAPI from '../../lib/atelierAPI.js';
 
-const ReviewComponent = () => {
+const ReviewComponent = (props) => {
+  const dispatch = useDispatch();
   const reviews = useSelector((state) => {
     return {
       reviews: state.reviews.reviewInfo,
       metadata: state.reviews.metadataInfo
     };
   });
```

Comment:
Add the productId to the returned object from the useSelector() hook, and then use the productId stored in the global state to dispatch your review action, instead of using the productId which you just prop drilled down a bunch of levels.

```suggestion
  const reviews = useSelector((state) => {
    return {
      productId: state.product.productId,
      reviews: state.reviews.reviewInfo,
      metadata: state.reviews.metadataInfo
    };
  });
```